src/registry_schemas/schemas/restoration.json

```diff
@@ -12,7 +12,10 @@
             "type": "object",
             "required": [
                 "date",
-                "type"
+                "type",
+                "offices",
+                "parties",
```

Comment:
do you know if for `limitedRestorationExtension` and `limitedRestorationToFull` that "offices" and "parties" is required?

Reply:
As per the design I assume it is [Design](https://preview.uxpin.com/306da47387722817e24fc77fd71476a1869a05fe#/pages/160258061/simulate/sitemap)

Reply:
I assume the same (for the same reason).